ipn/ipnlocal/local_test.go

```diff
@@ -2173,3 +2173,42 @@ func TestOnTailnetDefaultAutoUpdate(t *testing.T) {
 		})
 	}
 }
+
+func TestTCPHandlerForDst(t *testing.T) {
+	b := newTestBackend(t)
+	ip := netip.MustParseAddr("100.100.103.100")
+
+	tests := []struct {
+		dst         netip.AddrPort
+		wantHandler bool
```

Comment:
Comparing functions is unreliable, so I'm just looking to see whether we got a handler or not.